pkg/prompb/remote.proto

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016 Prometheus Team
```

Comment:
I'm not sure we should move the proto files in here. It is incorrect to try to recompile them into Go because of our optimizations.

Reply:
In that case, should I let them be as they were?